himl/interpolation.py

```diff
@@ -83,6 +84,15 @@ def resolve_secrets(self, data, default_aws_profile):
         return data
 
 
+class EnvVarResolver(object):
+    def resolve_env_vars(self, data):
+        # TO DO, ADD Logic for env_vars resolver
```

Comment:
what is this TODO referring to?

Reply:
Removed! Was added at first to track where changes are needed.